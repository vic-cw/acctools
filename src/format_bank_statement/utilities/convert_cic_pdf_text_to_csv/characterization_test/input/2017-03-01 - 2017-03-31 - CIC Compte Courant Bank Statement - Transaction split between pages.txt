                                                        Crédit Industriel et Commercial

                                                                                                 CIC PARIS SAINT MARTIN

                                                                                     224 RUE SAINT MARTIN 75010 PARIS
                                                ¤     u0820 82 34 56 (Service 0,12 /min + prix appel) FAX 01 44 16 81 39
                                                                                                                            23456@cic.fr  BIC : CMCIFRPP

                                                                                                                                                                                                      34567

                                                                                                                                                                                                      23456

                                                                                                           M  JOHN  DOE

                                                                                                           1  BROADWAY   AVENUE

                                                                                                           75001    PARIS

...........................................................................................................................V..O...T.R..E...C..O..N..S..E..I.L..L..E..R..:..M...S..é..b..a.s..t.i.e.n...D..U..P..O..N.T......................

                                          RELEVE ET INFORMATIONS BANCAIRES                                                                        Agence 23456

                                          31 mars 2017

                                                C/C CONTRAT PERSONNEL GLOBAL N° 00012345678 en euros (GD)

                                                TITULAIRE(S) : M JOHN DOE

                                          Date        Date valeur   Opération                                                      Débit euros    Crédit euros

                                                                    SOLDE CREDITEUR AU 28/02/2017                                                                                                     1.234,56                                                                                                  Pour tout appel en vue d'obtenir la bonne exécution d'un contrat ou le traitement

                                          01/03/2017  01/03/2017    RETRAIT DAB 2702 LCL PARIS ABCD                                       50,00

                                                                    LCL LE CREDIT LY CARTE 00123456                                                                                                                                              d'une réclamation, appelez le 09 69 32 06 06 appel non surtaxé

                                          01/03/2017  01/03/2017    PAIEMENT PSC 2802 MONTREUIL                                           11,28

                                                                    PHARMACIE ABCDE        CARTE 00123456

                                          02/03/2017  02/03/2017    VIR SEPA PROUT PROUT                                              500,00

                                          03/03/2017  03/03/2017    PAIEMENT CB      0203 MONTREUIL                                   12,40

KV.20170401.012345.6789.1011.12131 X 0 2                            BOUTIQUE CARTE 00123456

                                          (GE) : protégé par la Garantie de l'Etat                                                                <<Suite au verso>>

                                          (GD) : protégé par la Garantie des Dépôts

                                          www.garantiedesdepots.fr

                                                                                                                                                                                                      Page 1

                                            CREDIT INDUSTRIEL ET COMMERCIAL (CIC) - Banque régie par les articles L.511-1 et suivants du Code Monétaire et Financier - SA au capital de 608 439 888 

                                          6, avenue de Provence 75009 Paris - Adresse postale : 75452 Paris cedex 09 - tél 01 45 96 96 96 - swift CMCIFRPP - www.cic.fr - 542 016 381 RCS Paris - TVA intracommunautaire : FR34542016381

                                          M le Médiateur du CIC : 63 chemin Antoine Pardon 69160 Tassin la demi lune - Pour les opérations effectuées en sa qualité d'intermédiaire en opérations d'assurance : ORIAS 07 025 723 (www.orias.fr)
                                          RELEVE ET INFORMATIONS BANCAIRES                                                               Agence 23456

                                          31 mars 2017

                                                C/C CONTRAT PERSONNEL GLOBAL N° 00012345678                en  euros  (GD)

                                                TITULAIRE(S) : M JOHN DOE

                                          Date        Date valeur   Opération                                               Débit euros  Crédit euros

                                          07/03/2017  07/03/2017    PAIEMENT PSC 0503      MONTREUIL                        6,95

                                                                    SELLER NAME        CARTE 00123456

                                          08/03/2017  01/03/2017    PAIN IN THE ASS EXPENSE                                  9,70

                                          16/03/2017  16/03/2017    PAIEMENT PSC 1403 MONTREUIL                              0,60

KV.20170401.012345.6789.1011.12131 X 0 V                            OTHER SELLER CARTE 00123456

                                          22/03/2017  22/03/2017    RETRAIT DAB 2003       PLACE                             50,00

                                          (GE) : protégé par la Garantie de l'Etat

                                          (GD) : protégé par la Garantie des Dépôts

                                          www.garantiedesdepots.fr

                                                                                                                                         Page 2
                                                        Crédit Industriel et Commercial

                                                                                                    CIC PARIS SAINT MARTIN

                                                                                      224 RUE SAINT MARTIN 75010 PARIS
                                                     ¤  u0820 82 34 56 (Service 0,12 /min + prix appel) FAX 01 44 16 81 39
                                                                                                                            23456@cic.fr  BIC : CMCIFRPP

                                          RELEVE ET INFORMATIONS BANCAIRES                                                                              Agence 23456

                                          31 mars 2017

                                                C/C CONTRAT PERSONNEL GLOBAL N° 00012345678                            en  euros (GD)

                                                TITULAIRE(S) : M JOHN DOE

                                          Date          Date valeur  Opération                                                         Débit euros      Crédit euros

                                                                     DAB PLACE CARTE 00123456

                                          24/03/2017    24/03/2017   INCOMING      FUNDS                                                                                                               1.245,67

                                                                     REASON         /A01B23456789101

                                          27/03/2017    27/03/2017   PAIEMENT CB     2603 PLACE                                           13,00                                                                                                                                                                  Pour tout appel en vue d'obtenir la bonne exécution d'un contrat ou le traitement

                                                                     MERCHANT  CARTE 00123456

                                          27/03/2017    27/03/2017   PAIEMENT CB     2603 TRIFFOUILLY                                       17,59

                                                                     COCO MARCHANT  CARTE 00123456                                                                                                                                             d'une réclamation, appelez le 09 69 32 06 06 appel non surtaxé

KV.20170401.012345.6789.1012.13141 X 0 2  31/03/2017    31/03/2017   PAIEMENT CB     2803 PERPETTE                                         41,65

                                                                     ESCROC     CARTE 00123456

                                          Réf : 003                  SOLDE CREDITEUR AU 31/03/2017                                                                                                    12.345,67

                                          QXBAN : AB01 CMCI FRPP XXX CDE2 FGHI 3JKL MNOP 4QR                                IBAN : FR76 0123 4567 8910  1112 1314 151

                                            SITUATION DE VOS CREDITS en euros au 31 mars 2017

                                          Numéro                          Libellé du compte                                      Montant emprunté                                                     Solde (*)

                                          00012345678   ALLURE LIBRE                                                        EUR        +0,00                                                        +0,00

                                          00012345679   ETALIS                                                              EUR        +0,00                                                      +0,00

                                          (*) Solde correspondant à l'amortissement normal du prêt

                                          (GE) : protégé par la Garantie de l'Etat                                                                  <<Suite au verso>>

                                          (GD) : protégé par la Garantie des Dépôts

                                          www.garantiedesdepots.fr

                                                                                                                                                                                                      Page 3

                                            CREDIT INDUSTRIEL ET COMMERCIAL (CIC) - Banque régie par les articles L.511-1 et suivants du Code Monétaire et Financier - SA au capital de 608 439 888 

                                          6, avenue de Provence 75009 Paris - Adresse postale : 75452 Paris cedex 09 - tél 01 45 96 96 96 - swift CMCIFRPP - www.cic.fr - 542 016 381 RCS Paris - TVA intracommunautaire : FR34542016381

                                          M le Médiateur du CIC : 63 chemin Antoine Pardon 69160 Tassin la demi lune - Pour les opérations effectuées en sa qualité d'intermédiaire en opérations d'assurance : ORIAS 07 025 723 (www.orias.fr)
                                          RELEVE ET INFORMATIONS BANCAIRES                                                Agence 23456

                                          31 mars 2017

                                          AUTORISATIONS DE DECOUVERTS

                                          N° de compte              Libellé                                               Montant euros

                                          00012345678   DECOUVERT C. PERS. GLOBAL TAEG 15,12% AU  24/02/2017              0,00

                                          TOTAL MENSUEL DES FRAIS DEBITES SUR PRODUITS            ET SERVICES  BANCAIRES

                                          N° de compte              Libellé                                               Montant euros

                                          00012345678   TOTAL DES FRAIS EN FEV 2017                                       350.000,40

KV.20170401.012345.6789.1011.12131 X 0 V

                                          (GE) : protégé par la Garantie de l'Etat

                                          (GD) : protégé par la Garantie des Dépôts

                                          www.garantiedesdepots.fr

                                                                                                                          Page 4
