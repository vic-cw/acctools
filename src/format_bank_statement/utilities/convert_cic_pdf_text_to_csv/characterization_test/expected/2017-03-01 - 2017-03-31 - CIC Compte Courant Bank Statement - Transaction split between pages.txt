2017-03-01,2017-03-01,"RETRAIT DAB 2702 LCL PARIS ABCD LCL LE CREDIT LY CARTE 00123456","-50,00"
2017-03-01,2017-03-01,"PAIEMENT PSC 2802 MONTREUIL PHARMACIE ABCDE        CARTE 00123456","-11,28"
2017-03-02,2017-03-02,"VIR SEPA PROUT PROUT","-500,00"
2017-03-03,2017-03-03,"PAIEMENT CB      0203 MONTREUIL BOUTIQUE CARTE 00123456","-12,40"
2017-03-07,2017-03-07,"PAIEMENT PSC 0503      MONTREUIL SELLER NAME        CARTE 00123456","-6,95"
2017-03-08,2017-03-01,"PAIN IN THE ASS EXPENSE","-9,70"
2017-03-16,2017-03-16,"PAIEMENT PSC 1403 MONTREUIL OTHER SELLER CARTE 00123456","-0,60"
2017-03-22,2017-03-22,"RETRAIT DAB 2003       PLACE DAB PLACE CARTE 00123456","-50,00"
2017-03-24,2017-03-24,"INCOMING      FUNDS REASON         /A01B23456789101","1.245,67"
2017-03-27,2017-03-27,"PAIEMENT CB     2603 PLACE MERCHANT  CARTE 00123456","-13,00"
2017-03-27,2017-03-27,"PAIEMENT CB     2603 TRIFFOUILLY COCO MARCHANT  CARTE 00123456","-17,59"
2017-03-31,2017-03-31,"PAIEMENT CB     2803 PERPETTE ESCROC     CARTE 00123456","-41,65"
